backend/api/Services/RobotService.cs

```diff
@@ -373,5 +374,25 @@ private void NotifySignalROfUpdatedRobot(Robot robot, Installation installation)
         {
             _ = signalRService.SendMessageAsync("Robot updated", installation, robot != null ? new RobotResponse(robot) : null);
         }
+
+        public async Task<bool> IsRobotTelemetryWithinLimits(string robotId)
+        {
+            var robot = await ReadById(robotId);
+            if (robot is null) { return false; }
+            if (!robot.IsRobotPressureHighEnoughToStartMission())
+            {
+                logger.LogError("Robot with ID: {RobotId} cannot start missions because pressure value is too low.", robotId);
+                signalRService.ReporGeneralFailToSignalR(robot, $"Low pressure value for robot {robot.Name}", "Pressure value is too low to start a mission.");
+                return false;
+            }
+            if (!robot.IsRobotBatteryLevelHighEnoughToStartMissions())
+            {
+                logger.LogError("Robot with ID: {RobotId} cannot start missions because battery value is too low.", robotId);
+                signalRService.ReporGeneralFailToSignalR(robot, $"Low battery value for robot {robot.Name}", "Battery value is too low to start a mission.");
```

Comment:
The reason why i do it here is because I want to have a specific message for the pressure and another for the battery. And if this is done outside then I need to split this function into two and double the lines of code. So I was trying to optimize things. But I can also split it, let me know what you think after my explanation

Reply:
I suppose as long as it doesn't get too messy on the frontend in terms of alerts, then it is fine